specification/eventgrid/data-plane/Microsoft.HealthcareApis/stable/2018-01-01/HealthcareApis.json

```diff
@@ -221,7 +221,7 @@
         "VisionPrescription"
       ],
       "x-ms-enum": {
-        "name": "FhirResourceType",
+        "name": "HealthcareFhirResourceType",
```

Comment:
Should it be Healthcare or HealthCare?

Reply:
We have been using it as a single word, example: 
https://www.microsoft.com/en-us/industry/health/microsoft-cloud-for-healthcare 
https://www.hl7.org/fhir/overview.html

Reply:
Sounds good to me!